sdk/src/test/scala/com/horizen/account/performance/AccountForgeMessageBuilderPerfTest.scala

```diff
@@ -0,0 +1,127 @@
+package com.horizen.account.performance
+
+import com.horizen.SidechainTypes
+import com.horizen.account.fixtures.EthereumTransactionFixture
+import com.horizen.account.forger.AccountForgeMessageBuilder
+import com.horizen.account.history.AccountHistory
+import com.horizen.account.mempool.AccountMemoryPool
+import com.horizen.account.receipt.EthereumConsensusDataReceipt
+import com.horizen.account.receipt.EthereumConsensusDataReceipt.ReceiptStatus
+import com.horizen.account.state._
+import com.horizen.account.utils.{FeeUtils, ZenWeiConverter}
+import com.horizen.account.wallet.AccountWallet
+import com.horizen.block.MainchainBlockReferenceData
+import com.horizen.evm.interop.EvmLog
+import org.junit.Assert.assertEquals
+import org.junit.{Ignore, Test}
+import org.mockito.{ArgumentMatchers, Mockito}
+import org.scalatestplus.mockito.MockitoSugar
+import sparkz.core.NodeViewHolder.CurrentView
+
+import java.io.{BufferedWriter, FileWriter}
+import java.math.BigInteger
+import java.util.Calendar
+import scala.util.Try
+
+class AccountForgeMessageBuilderPerfTest extends MockitoSugar with EthereumTransactionFixture {
+
+  val stateView: AccountStateView = mock[AccountStateView]
+  Mockito
+    .when(
+      stateView.applyTransaction(
+        ArgumentMatchers.any[SidechainTypes#SCAT],
+        ArgumentMatchers.any[Int],
+        ArgumentMatchers.any[GasPool],
+        ArgumentMatchers.any[BlockContext],
+        ArgumentMatchers.any[Boolean]
+      )
+    )
+    .thenAnswer(asw => {
+      Try {
+        val gasPool = asw.getArgument(2).asInstanceOf[GasPool]
+        if (gasPool.getGas.compareTo(GasUtil.TxGas) < 0) {
+          throw GasLimitReached()
+        }
+        gasPool.subGas(GasUtil.TxGas)
+        new EthereumConsensusDataReceipt(2, ReceiptStatus.SUCCESSFUL.id, gasPool.getUsedGas, Array.empty[EvmLog])
+      }
+    })
+
+  val state: AccountState = mock[AccountState]
+  Mockito
+    .when(state.getBalance(ArgumentMatchers.any[Array[Byte]]))
+    .thenReturn(ZenWeiConverter.MAX_MONEY_IN_WEI) // Has always enough balance
+  Mockito.when(state.baseFee).thenReturn(BigInteger.ZERO)
+
+  Mockito.when(state.getNonce(ArgumentMatchers.any[Array[Byte]])).thenReturn(BigInteger.ZERO)
+  val mempool = AccountMemoryPool.createEmptyMempool(() => state)
+
+  val nodeView: CurrentView[AccountHistory, AccountState, AccountWallet, AccountMemoryPool] =
+    mock[CurrentView[AccountHistory, AccountState, AccountWallet, AccountMemoryPool]]
+  Mockito.when(nodeView.pool).thenReturn(mempool)
+
+  val blockContext = new BlockContext(Array.empty[Byte], 1000, BigInteger.ZERO, FeeUtils.GAS_LIMIT, 11, 2, 3)
+
+  @Test
+  @Ignore
+  def testComputeBlockInfo(): Unit = {
```

Comment:
Let's add a comment that explain the purpose of the test

Reply:
Done